lib/SyTest/Homeserver/Synapse.pm

```diff
@@ -1257,6 +1257,8 @@ sub generate_haproxy_map
 ^/_matrix/federation/v1/make_leave/                   federation_reader
 ^/_matrix/federation/v1/send_join/                    federation_reader
 ^/_matrix/federation/v1/send_leave/                   federation_reader
+^/_matrix/federation/(v1|v2)/make_knock/              federation_reader
+^/_matrix/federation/(v1|v2)/send_knock/              federation_reader
```

Comment:
I think only a v1 version exists? (Same goes for complement and the docs btw.)

Reply:
D'oh, good catch. Copy/paste fail.

Reply:
Although shouldn't the join, leave, etc ones include v2? 🤔

Reply:
Another good catch! I've corrected in https://github.com/matrix-org/sytest/pull/1382.